package-res/resources/web/pentaho/data/_filter/AbstractFilter.js

```diff
@@ -79,9 +79,20 @@ define([
       return _apply(this, dataTable);
     },
 
-    transform: function(iteratee){
+    /**
```

Comment:
Not that "visit" gives that idea, solely by the meaning of the word — as your statement seems to enunciate — it just looks like it is a common term for such an operation.